dmutils/email/dm_notify.py

```diff
@@ -18,11 +22,18 @@ def __init__(
             self,
             govuk_notify_api_key,
             govuk_notify_base_url='https://api.notifications.service.gov.uk',
-            logger=None
+            logger=None,
+            replacement_email_address=FROM_CURRENT_APP,
```

Comment:
This feels clearer if we name it something like `redirect_all_emails_to`.

I think the `FROM_CURRENT_APP` thing need a new name as it really doesn't make sense until you read the condition later on (`if current_app and replacement_email_address is FROM_CURRENT_APP else`). Not sure what a better name is though.

Reply:
Yes agreed on both counts

Reply:
I think the `FROM_CURRENT_APP` thing can just be None really. This sentinel is kinda confusing to me. I understand what it's there for, but I think with the renamed variable, `None` is a reasonable value to use and simpler to understand in the logic later on.